source/usage.rst

```diff
@@ -82,6 +82,24 @@ First, setup layman [#layman]_, then install rtrlib with the following commands:
 .. [#overlay] https://github.com/FRRouting/gentoo-overlay
 .. [#layman] https://wiki.gentoo.org/wiki/Layman
 
+
+Debian
+------
+
+Debian contains RTRlib since Buster [#DebianBuster]_ and can be installed with apt.
+The package is called ``librtr0``, for development you also need ``librtr-dev``.
```

Comment:
to be consistent with the general wording, please omit *you*, reword:

The package is called ``librtr0``, developers will require the ``librtr-dev`` package, too.

Reply:
done